openfisca_tasks/publish.mk

```
@@ -1,12 +1,26 @@
 .PHONY: build
 
+## Install project's build dependencies.
+install-dist:
+	@$(call print_help,$@:)
+	@python setup.py egg_info
+	@python -m pip install $$(grep -v "^\[" *.egg-info/requires.txt)
```

This is too much magic for me 😅
```suggestion
	@python -m pip install .[ci,dev]
```